packages/api/app/middleware.js

```diff
@@ -39,6 +49,21 @@ const handleBodyParserError = (res, error) => {
   }
 };
 
+/**
+* @param {number} minVersion -- Minimum supported version
+* @returns { RequestHandler } -- Request Handler
```

Comment:
```suggestion
* @param {number} minVersion - Minimum supported version
* @returns { RequestHandler } - Request Handler
```

The double dashes make the docs look weird when you hover over them in VS Code.